Generation/src/components/HepMCFileWriter.h

```diff
@@ -0,0 +1,35 @@
+#ifndef GENERATION_HEPMCFILEWRITER_H
+#define GENERATION_HEPMCFILEWRITER_H
+
+
+#include "GaudiAlg/GaudiAlgorithm.h"
+
+#include "FWCore/DataHandle.h"
+
+namespace HepMC {
+  class GenEvent;
+  class IO_GenEvent;
+}
+
+class HepMCFileWriter: public GaudiAlgorithm {
```

Comment:
It's only test samples that we have in HepMC which is sort of in line with my statement about "debugging" purposes ;-). We can certainly discuss whether those tests using these files should maybe be moved outside of the Examples.